### PR TITLE
Update overcommit topics for 4.0

### PR DESCRIPTION
**Pull request by mburke5678**
https://jira.coreos.com/browse/OSDOCS-256

### CODE REVIEW

**Comment by mburke5678** [comment]
@sjenning PTAL This is to remove node overcommit from the 4.0 docs, based on your comment here: https://github.com/openshift/openshift-docs/pull/12970#discussion_r250715475

File: modules/nodes-cluster-resource-configure-about.adoc [hunk 1]
```diff
@@ -28,7 +28,7 @@ Note the following about memory requests and memory limits:
 
   - The cluster administrator can assign quota or assign default values for the memory request value.
 
-  - The cluster administrator may override the memory request values that a developer specifies, in order to manage cluster overcommit. 
+//  - The cluster administrator may override the memory request values that a developer specifies, in order to manage cluster overcommit.
```
Comment (sjenning) [inline modules/nodes-cluster-resource-configure-about.adoc]: why just comment out rather than remove?

**Comment by sjenning** [comment]
@smarterclayton mentioned that he and @derekwaynecarr were looking not to carry the ClusterResourceOverride (CRO) forward into 4.0.

This PR is overly agressive though.  Documentation about LimitRanges and Pod requests/limits is still valid.  Only docs pertaining to CRO should be removed.

**Comment by mburke5678** [comment]
@sjenning I removed the section that referenced the `ClusterResourceOverride` parameter and updated a section that used the `node-config.yaml` file.

**Comment by mburke5678** [comment]
Jira for followup changes: https://jira.coreos.com/browse/OSDOCS-311

**Pull request merged**